content/updates/2020-08/2-API-search.md

```diff
@@ -0,0 +1,21 @@
+---
+pim_announcement_img: ./img/api-filter-attributes.png
+pim_announcement_alt_img: "API documentation about attribute filters"
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# New filters on product structure API endpoints
+::: meta-data type="New" features="API, Connectivity" available="late August" link-to-doc="https://api.akeneo.com/documentation/filter.html"
+:::
+
+With the new `search` parameter, you can now add filters on product structure endpoints: attributes, attribute groups, categories, and families.
```

Comment:
```suggestion
With the new `search` parameter, you can now add filters on product structure endpoints: attributes, attribute groups, categories, and families. This new parameter will help reducing the number of API calls needed to synchronize product data. 
```

Reply:
This is not exactly the role of these filters. But it will reduce the size of the API call responses and the number of data to process.